src/SFA.DAS.AssessorService.Data/RegisterQueryRepository.cs

```diff
@@ -255,5 +255,43 @@ public async Task<IEnumerable<OrganisationStandardPeriod>> GetOrganisatonStandar
                 return await connection.QueryAsync<OrganisationStandardPeriod>(sql);
             }
         }
+
+        public async Task<IEnumerable<AssessmentOrganisationSummary>> GetAssessmentOrganisationsByUkprn(string ukprn)
+        {
+            var connectionString = _configuration.SqlConnectionString;
+            if (!int.TryParse(ukprn, out int ukrpnNumeric))
+            {
+                return new List<AssessmentOrganisationSummary>();
+            }
+            using (var connection = new SqlConnection(connectionString))
+            {
+                if (connection.State != ConnectionState.Open)
+                    await connection.OpenAsync();
+                var assessmentOrganisationSummaries = await connection.QueryAsync<AssessmentOrganisationSummary>($@"select EndPointAssessorOrganisationId as Id, EndPointAssessorName as Name, EndPointAssessorUkprn as ukprn from [Organisations] where EndPointAssessorUkprn = {ukrpnNumeric}");
```

Comment:
Should these sql queries be parameterized and pass an obj instead of using string interpolation? 🍸

Reply:
yes agree, they should be paramaterized without fail for dapper.  Extra especially when they are strings.